logger/grpc_test.go

```diff
@@ -32,15 +39,15 @@ func TestGRPCLogger(t *testing.T) {
 	grpclog.Errorf("Errorf: %d\n", 1)
 
 	// Fatal tests are hard... assume they are good!
-	expect := `[INFO] Info,
-[INFO] Infoln
-[INFO] Infof: 1
-[WARN] Warning,
-[WARN] Warningln
-[WARN] Warningf: 1
-[ERR] Error,
-[ERR] Errorln
-[ERR] Errorf: 1
+	expect := `timeformat [INFO]  Info,
```

Comment:
This was actually the most interesting change. By using the `hclog.Logger` as the underlying implementation with a `*log.Logger` shim, we slightly change the spacing of the messages.

hclog will vertically align the prefixed level values so that the actual message always starts on the same column.